blueocean-web/src/main/js/main.jsx

```diff
@@ -26,7 +26,7 @@ class App extends Component {
             <div className="Site">
                 <div id="outer">
                     <header className="global-header">
-                        <ExtensionPoint name="jenkins.logo.top"/>
+                        <Extensions.Renderer name="jenkins.logo.top"/>
```

Comment:
Specifically `Extensions.Renderer` just refers to the component to render extensions. I wrote up docs based on the current set of changes: https://github.com/kzantow/blueocean-plugin/blob/01bf117c9f1bf792b13fa7b3be6e9f21ac5cbc68/js-extensions/EXTENSIONS.md#rendering-extension-points